<?xml version="1.0"?>
<launch>
    <!-- Pointcloud processing node -->
    <node name="path_following_perception_node"    pkg="path_following_perception"     type="path_following_node.py"    output="screen"/>
    <node name="CVOD_cfg"    pkg="feature_detection"     type="dynam_recfg.py"    output="screen"/>
    <node pkg="tf2_ros" type="static_transform_publisher" name="detection" args="0 0 0 0 0 0 zed2_left_camera_frame detection" />'
</launch>